wandb/sdk/lib/ipython.py

```diff
@@ -91,13 +93,13 @@ def update(self, value, label):
                 "Unable to render progress bar, see the user log for details"
             )
 
-    def close(self):
+    def close(self) -> None:
         if self._disabled or not self._displayed:
             return
         self._widget.close()
 
 
-def jupyter_progress_bar(min=0, max=1.0):
+def jupyter_progress_bar(min: float = 0, max: float = 1.0) -> Optional[ProgressWidget]:
```

Comment:
Union[int, float]?

Reply:
I assumed it should be float because it uses this:
https://github.com/wandb/client/blob/master/wandb/sdk/lib/ipython.py#L72
which by default calls this one:
https://github.com/jupyter-widgets/ipywidgets/blob/master/python/ipywidgets/ipywidgets/widgets/widget_float.py#L230

do you think we should also add `int`?